mavros_msgs/srv/MountConfigure.srv

```diff
@@ -0,0 +1,22 @@
+# MAVLink message: DO_MOUNT_CONTROL
+# https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONFIGURE
+
+std_msgs/Header header
+
+uint8 mode # See enum MAV_MOUNT_MODE.
+uint8 MAV_MOUNT_MODE_RETRACT = 0
+uint8 MAV_MOUNT_MODE_NEUTRAL = 1
+uint8 MAV_MOUNT_MODE_MAVLINK_TARGETING = 2
+uint8 MAV_MOUNT_MODE_RC_TARGETING = 3
+uint8 MAV_MOUNT_MODE_GPS_POINT = 4
```

Comment:
I think that you can drop `MAV_MOUNT_` prefix because it anyway will have namesapce `mavros::MountConfigure`.

Reply:
The name comes from the enumaration of the mavlink definitions: https://mavlink.io/en/messages/common.html#MAV_MOUNT_MODE_RETRACT Should I still simplify it?

Reply:
> Should I still simplify it?

Most times - Yes.
See https://github.com/mavlink/mavros/blob/master/mavros_msgs/msg/CommandCode.msg